src/pages/HomePage.tsx

```diff
@@ -114,167 +107,119 @@ const HomePage = () => {
-                                    <path d="M26.8309 13.3506V24.9893H29.7632H32.6891L32.7274 24.1776C32.753 23.7238 32.8041 22.2538 32.8424 20.8988C32.8807 19.5439 32.9318 18.355 32.9574 18.2592C32.9893 18.125 33.264 18.9878 34.1073 21.9023C34.7206 24.0051 35.2381 25.7499 35.2636 25.7755C35.3147 25.833 41.307 25.3601 41.3709 25.2961C41.3901 25.2706 40.7257 23.1806 39.8952 20.6368L38.3875 16.0222L38.707 15.741C39.7674 14.8142 40.6426 13.0246 40.8918 11.2542C41.0004 10.4808 40.9301 8.56339 40.764 7.85394C40.1252 5.07367 38.4323 2.91336 36.273 2.10805C35.3339 1.76291 35.4105 1.7693 30.9706 1.74373L26.8309 1.71817V13.3506ZM33.4174 7.22758C33.8709 7.45128 34.2415 8.0265 34.4395 8.81904C34.5737 9.34953 34.5545 11.0305 34.4076 11.5674C34.1329 12.5772 33.6473 13.1652 32.9191 13.3889L32.7721 13.4273V10.2315V7.02944L32.983 7.07418C33.0916 7.09336 33.2896 7.16366 33.4174 7.22758Z" fill="#666FE4" />
-                                    <path d="M41.0774 4.76085V7.73286H42.7064H44.3355V16.2974V24.8619H47.4019H50.4683V16.8087V8.75549H52.1293H53.7903V5.78348V2.81146L53.6498 2.7859C53.5156 2.75394 41.3329 1.78244 41.1604 1.78884C41.0966 1.78884 41.0774 2.42798 41.0774 4.76085Z" fill="#666FE4" />
-                                    <path d="M69.8881 14.0284V25.6288H73.0503H76.2126V14.0284V2.42797H73.0503H69.8881V14.0284Z" fill="#666FE4" />
-                                    <path d="M83.4328 3.23408C79.9703 3.41943 77.0891 3.57283 77.0316 3.57283C76.923 3.57922 76.9166 3.76457 76.9166 6.58319V9.58716H78.5457H80.1747V17.9919V26.3966H83.2731H86.3714V17.6404V8.8841H88.0963H89.8212V5.88013C89.8212 4.22475 89.802 2.88255 89.7765 2.88255C89.7445 2.88894 86.8953 3.04234 83.4328 3.23408Z" fill="#666FE4" />
-                                </svg>
-                            </div>
-                            <div className="flex-initial px-2 md:px-4">
-                                <svg width="20" height="13" viewBox="0 0 20 13" fill="none" xmlns="http://www.w3.org/2000/svg">
-                                    <path d="M17.7111 0.5L19.9717 2.79889L10.4717 12.5L0.97168 2.79889L3.23225 0.5L10.4717 7.89269L17.7111 0.5Z" fill="#666FE4" />
-                                </svg>
+            {/* {mediaSorted.length === 0 && <Loader />}
+            mediaSorted.length >= 0 && () */}
+            <div className="flex flex-col h-full lg:flex-row">
+                <Sidebar />
```

Comment:
should be removed. its a common for all pages so construct should be as it was. u overcomplicate u dont need to just follow what is there already and adjust inside in some places as structure is the same in design and site